hs_core/templates/pages/my-resources.html

```diff
@@ -188,120 +186,7 @@ <h4 class="panel-title">
                 </tr>
                 </thead>
                 <tbody>
-                    {% for res in collection %}
```

Comment:
generally concept is to add this into a separate template so that we can render it async for future calls